src/core/utils/url_store.ts

```diff
@@ -0,0 +1,38 @@
+/**
+* A place to store help URLs for error messages etc
+*/
+
+const url_store = {
+  base_url: "https://pusher.com",
+  urls: {
+    authentication_endpoint: {
+      path: "/docs/authenticating_users",
+    },
+    javascript_quick_start: {
+      path: "/docs/javascript_quick_start"
+    },
+  }
+}
+
+/** Builds a consistent string with links to pusher documentation
+*
+* @param {string} key - relevant key in the url_store.urls object
+* @return {string} suffix string to append to log message
+*/
+const buildLogSuffix = function(key: string) : string {
+  const url_prefix = "Check out:";
+  const url_obj = url_store.urls[key];
```

Comment:
I think we should try to enforce consistency by using camelcase everywhere in new code, even though there are parts that are currently inconsistent.